packages/vue-components/src/TipBox.vue

```diff
@@ -330,7 +304,7 @@ export default {
 
 <style scoped>
     .box-container {
-        width: 100%;
+        /* width: 100%; */
```

Comment:
Seems like this fixes the issue. Can remove it.

Reply:
OK will implement this then!